types/fs-extra/index.d.ts

```diff
@@ -254,7 +254,7 @@ export interface PathEntryStream {
     read(): PathEntry | null;
 }
 
-export type CopyFilter = ((src: string, dest: string) => boolean) | RegExp;
+export type CopyFilter = (src: string, dest: string) => boolean;
```

Comment:
Is this a bugfix, or a change from v4?

Reply:
Its a change from v4

https://github.com/jprichardson/node-fs-extra/blob/master/CHANGELOG.md

Reply:
Thanks. Could you back up the old version to a new folder as described in the readme? https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version